server/hot-reloader.js

```diff
@@ -186,6 +193,8 @@ export default class HotReloader {
     this.middlewares = [
       webpackDevMiddleware,
       webpackHotMiddleware,
+      queryParserMiddleware,
```

Comment:
I am not sure if we should add the `queryParserMiddleware` middleware here.
The `errorOverlayMiddleware` depends on a query being parsed.